### ISSUE TITLE
new rule: mapDispatchToProps-consistent-dispatch

### ISSUE THREAD

**Issue by DianaSuvorova** [DianaSuvorova/eslint-plugin-react-redux#24]
mapDispatchToProps-consistent-dispatch
This rule would warn the following case:

```js
const mapDispatchToProps = dispatch => ({
  onDoSomething: () => dispatch(toSomethingElse()),
  action1: action1
});
```
while below two are correct:

```js
const mapDispatchToProps = dispatch => ({
  onDoSomething: () => dispatch(toSomethingElse()),
  action1: () => dispatch(action1())
});
```

```js
const mapDispatchToProps = {
  onDoSomething: doSomethingElse
  action1,
});
```

as per discussion #10
